src/components/UserList.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import Search from './Search';
+import User from './User';
+
+
+class UserList extends Component {
+   
+    state = {
+        search: '',
+        loading: true,
+        users: []
+    };
+    componentWillReceiveProps(props, state) {
+
+    }
+    componentWillMount() {
+
+    }
+    componentDidMount() {
+        this.fetchData();
+    }
+    shouldComponentUpdate(nextProps, nextState) {
+        const { users, search, loading } = this.state;
+        return (
+            (JSON.stringify(users)) !== (JSON.stringify(nextState.users)) || search !== nextState.search ||
```

Comment:
What we are trying to achieve here? Do we really need to stringy the array, can it be done without this operation?

Reply:
Could we do the same with helper method, `indexOf`?